config/mail.php

```diff
@@ -223,22 +108,23 @@
                 'postmark',
             ],
         ],
+
     ],
 
     /*
     |--------------------------------------------------------------------------
     | Global "From" Address
     |--------------------------------------------------------------------------
     |
-    | You may wish for all e-mails sent by your application to be sent from
-    | the same address. Here, you may specify a name and address that is
-    | used globally for all e-mails that are sent by your application.
+    | You may wish for all emails sent by your application to be sent from
+    | the same address. Here you may specify a name and address that is
+    | used globally for all emails that are sent by your application.
     |
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDR', null),
-        'name' => env('MAIL_FROM_NAME', null),
+        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
```

Comment:
Changing the name of this `.env` var is going to cause a lot of problems - if we want to use the 'new' way going forward, can we do something like:

```php
env('MAIL_FROM_ADDRESS',env('MAIL_FROM_ADDR','hello@example.com'))
```

That way the old env var will still work - this could definitely be an issue for our hosted platform.

Reply:
(See how @jerm handled this in `MAIL_MAILER` and `MAIL_DRIVER` in this file, above)